### ISSUE TITLE
Cascading forge menu doesn't have the aria attributes placed at the correct elements

### ISSUE THREAD

**Issue by Ma1h01** [tyler-technologies-oss/forge#995]
Cascading forge menu is a menu button. Its aria attributes, `aria-haspopup` `aria-expanded` `aria-controls`, should be placed at the `button` tag. However, these aria attributes are added to the `forge-list-item` as shown in the image. The last section of this [page](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/) also suggests the activated popup should have a `menu` role.

<img width="1673" height="292" alt="Image" src="https://github.com/user-attachments/assets/00a1c877-2f23-4e78-a66e-d3537f913e07" />
